### PR TITLE
implemented resource sharing page for DWoC

### PR DESCRIPTION
**Pull request by vedikaag99**
## Issue that this pull request solves

 Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by vedikaag99** [comment]
https://user-images.githubusercontent.com/67699840/103291871-b3b85180-4a12-11eb-96ac-5c6dfdcd6852.mov

File: frontend/src/components/resource/resources.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
+  .resource-section {
+    background: #b6b6c2;
+    padding: 0 30px;
+  }
+  .resource-input {
```
Comment (Kajol-Kumari) [inline frontend/src/components/resource/resources.css:5]: please leave one line space at the end of each class in css file

File: frontend/src/components/resource/resources.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    }
+    .resource-parent {
+        display: block;
+        width: 100%;
+      }
+    
+      .child1 {
+        padding: 0 0 10px;
+      }
+  }
```
Comment (Kajol-Kumari) [inline frontend/src/components/resource/resources.css:129]: please leave one extra blank line at the end of the file

File: frontend/src/components/resource/resources.css [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  .resource-input i {
+    position: absolute;
+    right: 25px;
+    top: 40px;
+    font-size: 16px;
+    color: #000;
+  }
+  .resource-child {
+    /* display: flex; */
```
Comment (Kajol-Kumari) [inline frontend/src/components/resource/resources.css:68]: please remove the commented line

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@vedikaag99 design looks great! Just make these minor changes and it will be good to go :)

File: frontend/src/components/resource/resources.css (hunk 3 above)
Comment (vedikaag99) [reply frontend/src/components/resource/resources.css]: I have made the required changes.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Pull request merged**